### DISCUSSION TITLE
Section above comments for "updates", basically to add further updates, such as actions taken

### DISCUSSION THREAD

**Discussion by Savasdotexe** [Peppermint-Lab/peppermint#412] [Ideas]
I believe it would be useful to have a section just above the comment section to add "actions taken" or "pending steps" text.

To keep the minimalistic look, similar to being able to toggle being a public reply or an internal comment, just introducing buttons for "update" or "action taken" or other useful labels would suffice.

For example, ticket created for router not working. Update can be posted, with the action taken label that states for example router was restarted.

Here's an example I quickly drew up:

![image](https://github.com/user-attachments/assets/36aa0101-0382-48ba-900a-b1c33f91251c)

**Comment by potts99** [comment]
Great Idea, i was planning on building an "event store" that tracks updates etc. I am planning to add a close issue with x button, you'd be able to close with a comment, or in this case maybe a modal pop up to write up actions taken.

**Reply by Savasdotexe** [reply]
I mostly would like to use this for when I get an issue from a customer which is mostly over the phone and I create the ticket myself, I'd like to post updates and much prefer the order of posts consistent with actual timeline of events.

Looking at my mock up above, I think you could merge updates and comments together and have it in a nice timeline manner, with the labels of whether it's an action taken, a comment/update or public reply (something said to the customer), beside the username with background colours depending on the type of post for readability and **just in** case if needed a pin to the top for added benefit of anything like usernames or passwords provided by the customer for example.

**Reply by Savasdotexe** [reply]
You could move the close Issue with x to the right side bar and have it have its dropdown menu there, that's if I am understanding your idea correctly.

**Reply by potts99** [reply]
pretty much yeah :)
